language/move-prover/stackless-bytecode-generator/tests/testsuite.rs

```diff
@@ -31,6 +35,39 @@ fn get_tested_transformation_pipeline(
             pipeline.add_processor(Box::new(LiveVarAnalysisProcessor {}));
             Ok(Some(pipeline))
         }
+        "borrow" => {
```

Comment:
Would your live have been easier if we have a way to print each intermediate result of a step in the pipeline?

Reply:
I am not sure of that.  I try to explicitly remove information otherwise there is too much clutter in the printed output.